source/_patterns/02-organisms/global/site-header.mustache

```diff
@@ -4,11 +4,11 @@
       <a href="#maincontent" class="site-header__skip_to_content__link button button--default">Skip to Content</a>
     </div>
     <a href="{{homePagePath}}" class="site-header__logo_link">
-      <picture class="site-header__logo_link_image">
+      <picture>
         {{! 45.625em is 730px, which is $bkpt-site--medium }}
-        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-full.svg{{/assetRewrite}}" type="image/svg+xml" media="(min-width: 45.625em)">
-        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-symbol.svg{{/assetRewrite}}" type="image/svg+xml">
-        <img src="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-full-1x.png{{/assetRewrite}}" alt="eLife logo" class="site-header__logo_link"/>
+        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-xs.svg{{/assetRewrite}}" type="image/svg+xml" media="(min-width: 45.625em)">
+        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/journal-symbol.svg{{/assetRewrite}}" type="image/svg+xml">
```

Comment:
Where did the name change to `journal-symbol` come from? Curious why it's not a like-for-like replacement.

Reply:
It was what was supplied to me. I will change the name